### PR TITLE
Added the updated fields in the signup page using Material-UI 

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes Issue : #375 
signup-1-1-1

**Filling the signup form**

image

**creation of account in backend**

image

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@vaibhavdaren  Please review the PR

**Comment by jaskirat2000** [comment]
Looks cool. @vaibhavdaren will review this

File: src/auth/signup-form/signup-form.js [hunk 1]
```diff
@@ -9,70 +12,169 @@ class SignUpForm extends Component {
...
-  }
+    this.setState({ [e.target.name]:e.target.value })
+}
+
 
   onSubmit = (e) => {
     e.preventDefault();
-    this.props.registerUser(this.state, this.props.history);
+    if(this.state.password === this.state.cnfPassword){
+      const newUser = {
```
Comment (Rupeshiya) [inline src/auth/signup-form/signup-form.js:35]: destructure

Comment (tanujvyas10) [reply src/auth/signup-form/signup-form.js:36]: done

**Comment by tanujvyas10** [comment]
@vaibhavdaren all issues has been resolved 
please proceed

**Pull request merged**